RevenueCatUI/Views/IntroEligibilityStateView.swift

```diff
@@ -134,27 +134,6 @@ private extension IntroEligibilityStateView.Display {
 
 }
 
-// MARK: - Private
-
-@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
-private extension IntroEligibilityStateView.Display {
```

Comment:
https://github.com/RevenueCat/purchases-ios/commit/f16c3744fa4d5b999b89654cd29a75728605a347 was a bad rebase and duplicated this.